lib/embulk/filter/ruby_proc.rb

```diff
@@ -85,6 +87,13 @@ def self.transaction(config, in_schema, &control)
             eval(File.read(rowdef["proc_file"]), evaluator_binding, File.expand_path(rowdef["proc_file"]))
           end
         }.compact
+        @page_proc_store[transaction_id] = page_procs = task["pages"].map {|rowdef|
```

Comment:
I think that `rowdef` is not suiltable name for `pages` definition.

Reply:
Oops, sorry for the copy and paste code... Fixed it.